modules/appeals_api/spec/docs/legacy_appeals/v0_spec.rb

```diff
@@ -65,36 +65,8 @@ def swagger_doc
       response '422', 'Header Errors' do
         schema '$ref' => '#/components/schemas/errorModel'
 
-        describe 'X-VA-SSN and X-VA-File-Number both missing' do
-          let(:'X-VA-SSN') { nil }
-          let(:'X-VA-File-Number') { nil }
-
-          it_behaves_like 'rswag example',
-                          desc: 'when X-VA-SSN and X-VA-File-Number are missing',
-                          extract_desc: true,
-                          scopes:
-        end
-
```

Comment:
I decided to remove this example response where ICN is missing because RSwag actually converts this `nil` value to a query parameter like `/path?icn=`, where ICN is present but blank. This is not the same as a missing ICN, and the error message reflects that in the existing docs. As-is, it seems duplicative of the invalid length example. I do think it would be a good idea to include a true missing ICN example, but I could not figure out how to make rswag do that.